src/studio/ui/SettingsDialog.java

```diff
@@ -121,8 +150,18 @@ private void initComponents() {
 
         layout.setHorizontalGroup(
                 layout.createParallelGroup()
-                    .addComponent(chBoxShowServerCombo)
                     .addGroup(
+                            layout.createSequentialGroup()
+                                        .addComponent(lblLookAndFeel)
+                                        .addComponent(comboBoxLookAndFeel)
+                                        .addComponent(glue2)
+                        )
+                        .addGroup(
+                            layout.createSequentialGroup()
+                                        .addComponent(lblResultTabsCount)
+                                        .addComponent(txtTabsCount)
+                                        .addComponent(chBoxShowServerCombo)
```

Comment:
I think chBoxShowServerCombo should be on a separate line. Currently it appears on the same line as "Result tabs count" which suggests that they are related (the same way as Authentication, User and Password are related). But in reality they are not related at all.

Reply:
SUre